src/pyseekdb/utils/embedding_functions/openai_embedding_function.py

```
@@ -117,3 +117,50 @@ def _get_model_dimensions(self) -> dict[str, int]:
+        Args:
+            config: Dictionary containing the embedding function's configuration
+
+        Returns:
+            Restored OpenAIEmbeddingFunction instance
+
+        Raises:
+            ValueError: If the configuration is invalid or missing required fields
+        """
+        model_name = config.get("model_name")
+        if model_name is None:
+            raise ValueError("Missing required field 'model_name' in configuration")
```

Ruff (0.14.11)

Avoid specifying long messages outside the exception class

(TRY003)